database/network-database/scripts/loader.py

```diff
@@ -74,6 +79,11 @@ def LOAD_NETWORK():
             row_num += 1
     print('\\.')
 
-LOAD_SOURCES()
-LOAD_GENES()
-LOAD_NETWORK()
+# load_sources()
+Utils.load_sources(Constants.NETWORK_DATA_SOURCE, Constants.DATABASE_NAMESPACE)
+
+# load_genes()
+Utils.load_genes(Constants.GENE_SOURCE, Constants.DATABASE_NAMESPACE, False)
+
+# load_network()
+Utils.load_network(Constants.NETWORK_SOURCE, Constants.DATABASE_NAMESPACE, False)
```

Comment:
With _utils_ now included, we can see that the argument is `is_protein` so we want these to be:
```suggestion
Utils.load_genes(Constants.GENE_SOURCE, Constants.DATABASE_NAMESPACE, is_protein=False)

# load_network()
Utils.load_network(Constants.NETWORK_SOURCE, Constants.DATABASE_NAMESPACE, is_protein=False)
```